features/webtransport.yml

```diff
@@ -3,27 +3,6 @@ description: The `WebTransport` API transmits data between a client and a server
 spec: https://w3c.github.io/webtransport/
 caniuse: webtransport
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/3472
-# Status override because createUnidirectionalStream() is marked as partial
-# implementation in Firefox in BCD.
 status:
-  baseline: false
-  support:
-    chrome: "97"
-    chrome_android: "97"
-    edge: "97"
-    firefox: "114"
-    firefox_android: "114"
-# BYOB reader support arrived later than the rest
-# - api.WebTransport.byob_readers
-# - api.WebTransport.createUnidirectionalStream.byob_readers
-# - api.WebTransportDatagramDuplexStream.byob_readers
-
-# As of 2023-08-10, the spec calls these "feature at risk"
-# - api.WebTransport.WebTransport.options_congestionControl_parameter
-# - api.WebTransport.congestionControl
-
-# TODO: figure out what the story is with WebTransportReceiveStream
-# This feature has a different compatibility story than closely-related
-# interfaces (i.e., WebTransportBidirectionalStream)
-# - api.WebTransportReceiveStream
-# - api.WebTransportReceiveStream.getStats
```

Comment:
Can we preserve these comments or migrate them to an issue? It looks like these keys didn't make into the tag in BCD, so this will just evaporate without being dealt with.

Reply:
I've restored these comments.